psij-ci-run

```diff
@@ -16,7 +16,10 @@ failifempty() {
     fi
 }
 
-source psij-ci-load
+if [ -f psij-ci-load ]; then
+    source psij-ci-load
+fi
```

Comment:
I'm not completely sold on keeping the invocation of `psij-ci-load`. Any user who put anything in their psij-ci-load is going to get a conflict when they pull this commit (I think) and so should figure out that something has changed. How about putting a warning message instead? Or keeping what you have but also echoing a deprecation warning.

Reply:
I'm thinking that any user who has a working setup with something in `psij-ci-load` can and should probably keep it as is.

That said, I'm not completely sold on keeping it either, but for a slightly different reason: if we need to instruct a user to do some fancy setup requiring something like `psij-ci-load`, we could as well instruct them to put the relevant lines in `psij-ci-run`.

Reply:
So maybe there's consensus here that this should be removed.